File: Ejercicio_2/ejercicio2.py
# Escriba un algoritmo en pseudo código, para verificar si un número es par o impar.

# Funcion para determinar la paridad de un numero

def even_or_odd(n):
    if n % 2 == 0:
        print("\nEl numero es par!\n")
    else:
        print("\nEl numero es impar!\n")


num = float(input("\nIngrese un numero: "))

even_or_odd(num)

File: Ejercicio_2/ejercicio2.c
// Escriba un algoritmo en pseudo código, para verificar si un número es par o impar.

#include <stdio.h>

#

void even_or_odd(int num)
{
    if (num % 2 == 0)
        printf("\nEl numero es par!\n");
    else
        printf("\nEl numero es impar!\n");
}

void main()
{
    int num;
    printf("\nIngrese el numero: ");
    scanf("%d", &num);
    even_or_odd(num);
}